### PR TITLE
2023 updates

### PR DESCRIPTION
**Pull request by WeirdRubberDuck**
Clarify criteria for transfer function

### CODE REVIEW

**Comment by WeirdRubberDuck** [comment]
I don't understand why the first two commits are there. These should already be in master?

File: instructions/scivis.tex [hunk 1]
```diff
@@ -83,11 +83,12 @@ \section{The Tasks}
...
   \item Using the front-to-back compositing method, use your transfer function editor to interactively manipulate the transfer function to show the lab assisstant what is on the inside of the dataset.  Your editor must be capable of the following:
   \begin{todolist}
-    \item Have a single transfer function editor
     \item Change the transfer function without needing to reload the webpage
-    \item Have the ability to change the color and opacity mapping for volume samples
-    \item Be able to interact with the editor and change the transfer function settings with the mouse
-    \item Have the ability to simulate isosurface rendering by being able to create ``spiky'' transfer functions that have a high opacity only for a few values with different colors.
+    \item Have the ability to change the color and opacity mapping for volume samples, to any color
+    \item Be able to interact with the editor and change the transfer function settings using the mouse
+    \item Have the ability to simulate isosurface rendering by being able to create "spiky" transfer functions that have a high opacity only for a few values with different colors
```
Comment (alexanderbock) [inline instructions/scivis.tex]: "spiky" is being highlighted on GitHub for some reason?

**Review by alexanderbock — APPROVED** [review]

File: instructions/scivis.tex (hunk 1 above)
Comment (WeirdRubberDuck) [reply instructions/scivis.tex]: Probably due to the quotations. I'll use the ` ``spiky'' ` version instead

Comment (WeirdRubberDuck) [inline instructions/scivis.tex]:
```suggestion
    \item Have the ability to simulate isosurface rendering by being able to create ``spiky'' transfer functions that have a high opacity only for a few values with different colors
```

**Comment by WeirdRubberDuck** [comment]
Ready for merge (I don't have the rights to do it)

**Pull request merged**